namespaces/live-1.cloud-platform.service.justice.gov.uk/calculate-release-dates-api-dev/00-namespace.yaml

```diff
@@ -0,0 +1,16 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: calculate-release-dates-api
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/slack-channel: "calculate_release_dates"
+    cloud-platform.justice.gov.uk/slack-alert-channel: "calculate_release_dates_dev"
+    cloud-platform.justice.gov.uk/application: "Calculate Release Dates API"
+    cloud-platform.justice.gov.uk/owner: "Digital Prison Services: dps-hmpps@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/hmpps-template-kotlin.git"
+    cloud-platform.justice.gov.uk/team-name: "farsight-devs"
```

Comment:
The source code wants to be the name it will be, not the template. So `calculate-release-dates-api.git` here please.

Reply:
I've changed that